lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -13,17 +14,23 @@ export default function DashboardApp() {
 
   return (
     <div className="flex flex-col min-h-screen gap-5">
-      <div
-        className={classnames(
-          'flex justify-center p-3 w-full',
-          'bg-white border-b shadow',
-        )}
-      >
-        <img
-          alt="Hypothesis logo"
-          src="/static/images/hypothesis-wordmark-logo.png"
-          className="h-10"
-        />
+      <div className="px-3 py-4 bg-white border-b shadow">
+        <div className="flex justify-between items-center mx-auto max-w-6xl">
+          <LogoIcon />
```

Comment:
The logo icon should be something you can click on to take you to some kind of "home" view. I suggest that should be the "All courses" view at the moment. In future there might be a different view that is more appropriate as a home view.

Reply:
Good point